### PR TITLE
📝 [RUMF-1397] Use new init parameters in the doc

### PR DESCRIPTION
**Pull request by bcaudan**
## Motivation

Following #1917, document new parameters

## Changes

Document `sessionSampleRate`, `traceSampleRate` and `trackUserInteractions`

## Testing

N/A

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: packages/rum/README.md [hunk 1]
```diff
@@ -73,9 +73,9 @@ datadogRum.init({
   //  service: 'my-web-application',
   //  env: 'production',
   //  version: '1.0.0',
-  sampleRate: 100,
+  sessionSampleRate: 100,
   premiumSampleRate: 100, // if not included, the default is 100
-  trackInteractions: true,
+  trackUserInteractions: true,
```
Comment (BenoitZugmeyer) [inline packages/rum/README.md:78]: **🔨 warning:** ‏You changed the doc corresponding to code snippet to be used before v4.20.0 and v4.10.2. It should stay the same here.


**💬 suggestion:** ‏maybe add a similar section "before v4.30.0"

File: packages/logs/README.md [hunk 2]
```diff
@@ -127,7 +127,7 @@ The following parameters are available to configure the Datadog browser logs SDK
...
 | `forwardConsoleLogs`  | `"all"` or an Array of `"log"` `"debug"` `"info"` `"warn"` `"error"`      | No       | `[]`            | Forward logs from `console.*` to Datadog. Use `"all"` to forward everything or an array of console API names to forward only a subset.                                                |
 | `forwardReports`      | `"all"` or an Array of `"intervention"` `"deprecation"` `"csp_violation"` | No       | `[]`            | Forward reports from the [Reporting API][8] to Datadog. Use `"all"` to forward everything or an array of report types to forward only a subset.                                       |
-| `sampleRate`          | Number                                                                    | No       | `100`           | The percentage of sessions to track: `100` for all, `0` for none. Only tracked sessions send logs.                                                                                    |
```
Comment (BenoitZugmeyer) [inline packages/logs/README.md:130]: **💬 suggestion:** ‏keep the deprecated option here, similarly to RUM docs?

**Review by BenoitZugmeyer — APPROVED** [review]

File: packages/rum/README.md [hunk 3]
```diff
@@ -102,7 +125,7 @@ datadogRum.init({
 
 </details>
 
-The `trackInteractions` and `trackFrustrations` parameters enable the automatic collection of user clicks in your application. **Sensitive and private data** contained on your pages may be included to identify the elements interacted with.
+The `trackUserInteractions` and `trackFrustrations` parameters enable the automatic collection of user clicks in your application. **Sensitive and private data** contained on your pages may be included to identify the elements interacted with.
```
Comment (rtrieu) [inline packages/rum/README.md:128]:
```suggestion
The `trackUserInteractions` and `trackFrustrations` parameters enable the automatic collection of user clicks in your application. **Sensitive and private data** contained in your pages may be included to identify the elements interacted with.
```
I believe "contained in" is more grammatically correct.

File: packages/rum/README.md [hunk 4]
```diff
@@ -197,7 +251,7 @@ Add the generated code snippet to the head tag of every HTML page you want to mo
...
```
Comment (rtrieu) [inline packages/rum/README.md:254]:
```suggestion
The `trackUserInteractions` and `trackFrustrations` parameters enable the automatic collection of user clicks in your application. **Sensitive and private data** contained in your pages may be included to identify the elements interacted with.
```

File: packages/rum/README.md [hunk 5]
```diff
@@ -276,7 +355,7 @@ Add the generated code snippet to the head tag (in front of any other script tag
...
```
Comment (rtrieu) [inline packages/rum/README.md:358]:
```suggestion
The `trackUserInteractions` and `trackFrustrations` parameters enable the automatic collection of user clicks in your application. **Sensitive and private data** contained in your pages may be included to identify the elements interacted with.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]

**Review by rtrieu — APPROVED** [review]

**Pull request merged**